docs/ecctl-getting-started.asciidoc

```diff
@@ -44,6 +44,32 @@ Updating Homebrew...
 
 NOTE: To get autocompletions working make sure to run `source <(ecctl generate completions)`. If you prefer to add to your shell interpreter configuration file run `echo "source <(ecctl generate completions)" >> ~/.bash_profile` or `echo "source <(ecctl generate completions)" >> ~/.zshrc`.
 
+[float]
+[id="{p}-upgrading-macos"]
+=== Upgrade on macOS
+
+To upgrade {p} via brew, it's very simple to do so:
+
+[source]
+----
+$ brew upgrade ecctl
+
```

Comment:
If the output isn't essential (i.e. you don't need to glean some information from it to use in a subsequent step), you could omit lines 56 - 70. Less is often more.